### DISCUSSION TITLE
Feature Request: add vim and emacs bindings

### DISCUSSION THREAD

**Discussion by aldevv** [ueberdosis/tiptap#5546] [Feature Requests]
### Description

I wish there was a way to add vim and emacs bindings easily

### Use Case

whenever I work in an editor I prefer using vim bindings

### Type

New feature

**Comment by nperez0111** [comment]
https://tiptap.dev/docs/editor/core-concepts/keyboard-shortcuts#overwrite-keyboard-shortcuts

You can create your own key bindings. But this is way out of scope to be built into tiptap
